packages/next/src/cli/next-build.ts

```diff
@@ -20,8 +20,8 @@ type NextBuildOptions = {
 }
 
 const nextBuild = (options: NextBuildOptions, directory?: string) => {
-  process.on('SIGTERM', () => process.exit(0))
-  process.on('SIGINT', () => process.exit(0))
+  process.on('SIGTERM', () => process.exit(128 + 15))
+  process.on('SIGINT', () => process.exit(128 + 2))
```

Comment:
```suggestion
  process.on('SIGTERM', () => process.exit(143))
  process.on('SIGINT', () => process.exit(130))
```
I think we can explicitly put the exit codes since they are known.

Reply:
Done, thanks. I originally separated 128 to make it more clear, but it's fine either way.